iroha/src/crypto.rs

```diff
@@ -5,26 +5,22 @@ use ursa::{
         digest::{Input, VariableOutput},
         VarBlake2b,
     },
-    keys::PublicKey as UrsaPublicKey,
-    signatures::{ed25519::Ed25519Sha512, SignatureScheme},
+    keys::{PrivateKey, PublicKey as UrsaPublicKey},
```

Comment:
Maybe rename `PrivateKey` to `UrsaPrivateKey` to be consistent with `UrsaPublicKey` import

Reply:
I think we need to add our PrivateKey first. Let's discuss it in RFC
https://wiki.hyperledger.org/display/iroha/Encapsulation+of+crypto+related+functionality